src/panel/poi/PoiPanel.jsx

```diff
@@ -180,7 +178,10 @@ export default class PoiPanel extends React.Component {
   backToList = () => {
     Telemetry.add(Telemetry.POI_BACKTOLIST);
     fire('restore_location');
-    window.app.navigateTo(`/places/?type=${this.props.sourceCategory}`);
+    const uri = this.props.resource.category
```

Comment:
I'm not 100% sure `q` and `category` can't be present at the same time. And anyways, as we are going to more genericity of this feature, with more possible paramers, it would be safer to have a function that generates the query string based on all that is present in the `resource` object.

Reply:
ok, I added a simple loop to add params